gridt/src/app/settings/settings.page.html

```diff
@@ -0,0 +1,177 @@
+<ion-header>
+  <ion-toolbar >
+    <ion-buttons slot="start">
+      <ion-menu-button></ion-menu-button>
+    </ion-buttons>
+    <ion-title>
+      Settings
+    </ion-title>
+  </ion-toolbar>
+</ion-header>
+
+
+<ion-content padding>
+  <!-- Refreshes page if pulled down (mobile only)-->
+  <ion-refresher slot="fixed" (ionRefresh)="refreshPage($event)" class="ion-hide-md-up">
+    <ion-refresher-content></ion-refresher-content>
+  </ion-refresher>
+
+  <ion-grid>
+    <ion-row justify-content-center>
+      <ion-col size="12" size-sm="8" offset-sm="2" size-md="6" offset-md="3" text-center>
+        <ion-card>
+          <ion-card-header>
+            <ion-card-title>Profile</ion-card-title>
+          </ion-card-header>
+
+          <ion-card-content>
+            <ion-list>
+              <ion-item class="ion-text-center" lines="none">
+                <ion-avatar class="ion-text-center">
+                  <img [src]="gravatar">
```

Comment:
When the server is unavailable this has a weird effect of creating a call to `/null`. Consider wrapping this in an if statement.

Reply:
Is it not the case that the settings always return _something_? In that case this would do nothing. Writing a test for this would not be a bad idea.